app/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx

```diff
@@ -79,6 +79,13 @@ export const FilterBar: React.FC<FilterBarProps> = ({
           optionLabelName="label"
           {...{ filterValue, onChange }}
         />
+        <FilterSelect
```

Comment:
Do we have any tests for the `FilterBar`? Can we add some tests there too, in addition to the `filterExperiments.test.tsx`?

Reply:
oh `filterBar` shows test cases so that everything is selected, and `filterExperiments.test.tsx` all the other cases like one targeting is selected, multiple targeting is selected and no targeting is selected and default - all targeting. I have covered all the scenarios, do you think I am missing something 🤔

Reply:
wowowow that FilterBar test is smalllll 😂 Your `filterExperiments` tests cover the cases, this is fine!